### ISSUE TITLE
Klipper crashing during input shaper

### ISSUE THREAD

**Issue by caall99** [xblax/flashforge_adm5_klipper_mod#105]
Cannot perform input shaper on v00.04 beta with klipper screen and wifi connection. It crashes after testing x axis and cannot start testing y axis before crashing. When I switched from wifi to ethernet I was able to complete input shaping. Guessing overhead on wifi is too high?

**Comment by consp** [comment]
Could also be something else. Do you have the logs available from `/var/log/KlipperScreen.log` ?

**Comment by consp** [comment]
> How do I access that folder? I cannot get into file structure with WinSCP.

No longer needed, I got the error on my version as well.

**Comment by caall99** [comment]
Can you still educate me on SSHing into file structure through winscp? What port?

**Comment by caall99** [comment]
I see your instructions now. Thank you very much!

**Comment by xblax** [comment]
Renamed the issue since this problem was observed before, it's not related to v00.04 beta.
I also suspect that it's not related to WIFI, it might be a general issues with available system resources or a bug in Klipper.

Closing this issue as duplicate of https://github.com/xblax/flashforge_adm5_klipper_mod/issues/64
